# Assignment

Add your name to one of the rows in this spreadsheet [here](https://docs.google.com/spreadsheets/d/1jYOx_BCUWnR7IhMTmw4oaymGovGxao5-P4RBRpjKdDA/edit#gid=356692471). The genes marked in the row next to your name are the ones you need to analyse for the assignment.


You need to make a concatenated dataset of the 4 genes that have been assigned to you, along with a gene partition file (as you did in [Tutorial 2](../Tutorials/2.Alignments)), then analyse this dataset in IQ-TREE (as you did in [Tutorial 3](../Tutorials/3.ModelSelection) and [Tutorial 5](../Tutorials/5.MaximumLikelihood)), and finally in BEAST (as you did in [Tutorial 6](../Tutorials/6.TimingDivergence)).

Make a short report including the resulting maximum likelihood tree with branch support as well as the BEAST tree showing the estimated times of divergence for the different lineages. In your report also answer the following three questions. Please include your alignment together with the report.

1) How does the alignment for the rRNA (12s and 16S are rRNA mitochondrial genes) gene looks compared to the protein-coding genes in your dataset?

2) Are there any differences between the maximum likelihood tree based on these four genes compared to the tree you inferred in Tutorial 5 based on a different set of four genes? If there are differences, what do you think is the reason for them?

3) If there are differences in topology, illustrate them in the trees using FigTree as described in [Tutorial 7](../Tutorials/7.TreeManipulation).

4) Are there any differences between the times of divergence inferred from these four genes compared to the tree you inferred in Tutorial 6?

If you have any questions about the assignment, feel free to email us.

The report is due on Friday, May 9 by midnight, and it is being graded on the pass/fail scale.
